### PR TITLE
facets: expand facet items by link

### LINKED ISSUE #87: Facets: add a "more" link or button.
The "Author" facet is limited to 5 entries even if there are more authors for the results.
It could offer more entries by the use of a "more" button at the bottom of the facet, for example like on explore.rero.ch.

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Adds a link to see more or less results on the facet.
* Closes #87

How to test:
- `script/bootstrap`
- Search in documents
- Check link on facets (see only if the facet results > 10)

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

### CODE REVIEW

File: rero_ils/config.py [hunk 1]
```diff
@@ -718,18 +718,48 @@ def _(x):
     },
 }
 
+# Number of results in facet
+RERO_ILS_AGGREGATION_SIZE = 50
```
Comment (jma) [inline rero_ils/config.py]: `RERO_ILS` prefix can be removed as it is a local variable.

File: ui/src/app/records/search/search.component.html [hunk 2]
```diff
@@ -28,21 +28,41 @@ <h6>
...
-        <ul class="list-unstyled collapse" [ngClass]="{'show': startOpen(item.title)}" id="{{'agg_'+item.title }}">
-          <li class="form-check" *ngFor="let bucket of item.buckets">
-            <input class="form-check-input" type="checkbox" [checked]="isFiltered(item.title, bucket.key)" (click)="aggFilter(item.title, bucket.key) ">
-            <label class="form-check-label">
-              <span *ngIf="bucket.name">{{ bucket.name | translate }}</span>
-              <span *ngIf="!bucket.name">{{ bucket.key | translate }}</span> ({{ bucket.doc_count }})
-            </label>
-          </li>
-        </ul>
+        <div class="collapse" [ngClass]="{'show': startOpen(item.title)}" id="{{'agg_'+item.title }}">
```
Comment (jma) [inline ui/src/app/records/search/search.component.html]: Can be probably replaced by a splice pipe: https://angular.io/api/common/SlicePipe. And a component for each bucket.

File: rero_ils/utils.py [hunk 3]
```diff
@@ -65,3 +65,25 @@ def send_mail(subject, recipients, template, language, **context):
 def unique_list(data):
     """Unicity of list."""
     return list(dict.fromkeys(data))
+
+
+def get_agg_config(index_name, field):
+    """Get Elasticsearch aggragation term configuration.
```
Comment (iGormilhit) [inline rero_ils/utils.py]: s/aggragation/aggregation/

All occurences in this file.

Comment (Garfield-fr) [reply rero_ils/utils.py]: done

**Pull request merged**